serpentTools/parsers/depletion.py

```diff
@@ -47,17 +47,20 @@ def __init__(self, filePath):
         #  TOT_ADENS --> ('ADENS', )
         #  ING_TOX --> ('ING_TOX', )
 
+        # track how many materials appear in the file
+        self.materialCount = 0
```

Comment:
Since this is only used in the `_precheck` method, this doesn't need to be an instance attribute. Could be confusing if `self.materialCount != len(self.materials)`

Reply:
done!